src/mongo/client/bulk_operation_builder.h

```diff
@@ -68,8 +68,12 @@ class MONGO_CLIENT_API BulkOperationBuilder {
      * @param client The connection to use.
      * @param ns The namespace to apply the operations to.
      * @param ordered Whether or not ordering matters for these operations.
+     * param bypassDocumentValidation Whether to bypass document validation for these operations,
      */
-    BulkOperationBuilder(DBClientBase* const client, const std::string& ns, bool ordered);
+    BulkOperationBuilder(DBClientBase* const client,
+                         const std::string& ns,
+                         bool ordered,
+                         bool bypassDocumentValidation);
```

Comment:
Maybe default the new parameter to 'false' to avoid source level breakage? That or overload the function.